overrides/config/ftbquests/quests/chapters/hard_eyes.snbt

```diff
@@ -95,7 +95,25 @@
 				"0F399B0FB108E920"
 				"5B4CB0F76C78E90B"
 			]
-			description: ["&4Requires Nether Access&r"]
+			description: [
+				"&4Requires Nether Access&r"
+				""
+				"• Wildfire Crown - Crafted with 5 &eWildfire Crown Pieces&r"
+				""
+				"• Mutant Blaze Core - Dropped by &4Mutant Blazes&r"
+				""
+				"• Netherite Scrap - Mined in The Nether depths or found in Bastion chests"
+				""
+				"• Hone: Gild III -  Crafted with weaker &eHone: Gild&r scrolls"
+				""
+				"• Crimson Mosquito Larva - Crafted with a &eMaggot &rand a &eCrimson Mosquito Probuscis&r"
```

Comment:
There is "Mosquito Proboscis" from "Mosquito Probuscis" here

Reply:
AHHH